packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -171,3 +205,60 @@ const removeAddedModuleCode = (modulePath: string, index: number) => {
   sfp.removeUnusedImports();
   sfp.save();
 };
+
+/**
+ * Returns the postcss.config filepath if it exists in module.
+ * Else returns the root postcss.config filepath.
+ * If there is none, throws error.
+ *
+ * @param rootFolders
+ * @param subfolders
+ * @param filename of module
+ * @returns string
+ */
+const getPostcssConfigFilepath = (
+  rootFolders: any,
+  subfolders: any,
+  filename: string
+): string => {
+  for (const extension of postcssExtensions) {
+    const filePath = path.join(
+      rootFolders.source,
+      subfolders.modules,
+      `${filename}/postcss.config.${extension}`
+    );
+    if (fs.existsSync(filePath)) {
+      return filePath;
+    }
+  }
+
+  for (const extension of postcssExtensions) {
+    const filePath = path.join(`postcss.config.${extension}`);
+    if (fs.existsSync(filePath)) {
+      return filePath;
+    }
+  }
+
+  throw new Error(`Cannot find a postcss.config file for ${filename}`);
```

Comment:
A user doesn't need to use TW/PostCss at all.

Reply:
Ah oops.